app/app/routes.py

```diff
@@ -0,0 +1,23 @@
+from flask import Blueprint, render_template, request, redirect, url_for, flash, jsonify
+from flask_login import login_required, current_user
+from . import db
+from .models import  User
+
+
+routes = Blueprint('routes', __name__)
+
+@routes.route('/')
+#login_required
+def home():
+    if current_user.is_authenticated:
+        user = User.query.filter_by(id = current_user.id).first()
+
+    return render_template('home.html', user=current_user)
+
+@routes.route('/analytics')
+#login_required
+def data():
+    if current_user.is_authenticated:
+        user = User.query.filter_by(id = current_user.id).first()
+
+    return render_template('data.html', user=current_user)
```

Comment:
are we building a server-side application or a client-side application?

(just a question). There's 2 ways to do this:
1. Client-side application where there is no refresh between interaction (uses AJAX to contact Flask API_
2. Server-side application where there is refresh between interaction (always render html, css for every database interaction)